src/main/kotlin/systems/ajax/motrechko/airguardian/controller/DroneController.kt

```diff
@@ -23,29 +26,25 @@ class DroneController(
     private val droneService: DroneService
 ) {
     @GetMapping
-    fun getAllDrone(): ResponseEntity<List<DroneResponse>> = ResponseEntity.ok(
-        droneService.getAllDrones().toResponse()
-    )
+    fun getAllDrone(): Flux<DroneResponse> =
```

Comment:
how does it look for a client? One response with a list of DroneResponse or many responses?
If many responses, then consider changing the return type to Mono<List<DroneResponse>> as the goal of this endpoint is to return all elements and complete